includes/WikiPage.php

```diff
@@ -590,9 +590,11 @@ public function isCountable( $editInfo = false ) {
 	 *
 	 * The target will be fetched from the redirect table if possible.
 	 * If this page doesn't have an entry there, call insertRedirect()
+	 *
+	 * @param bool $userMaster
 	 * @return Title|mixed object, or null if this page is not a redirect
 	 */
-	public function getRedirectTarget() {
+	public function getRedirectTarget( $userMaster = false ) {
```

Comment:
Yea, it is pretty unintuitive. This file also has a method which accepts the string `"fromdbmaster"` to do the same thing which is one of the uglier things I've seen in a while. Good to be consistent though, I'll update to use `\Title::GAID_FOR_UPDATE`.